pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -109,7 +109,7 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "zoom";
-  inherit version;
+  version = versions.${system} or throwSystem;
```

Comment:
Same here

Reply:
Besides the reason I wrote above, evaluating `versions.${system}` down here is more consistent with how `srcs.${system}` is also evaluated down here inside the mkDerivation call rather than up in the let bindings.